calculator_api_server/schemas.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+
+class CalculationResult(BaseModel):
+    result: float
+
+
+class CommonResponseModel(BaseModel):
+    data: CalculationResult
```

Comment:
```suggestion
    data: BaseModel
```

`CommonResponseModel`인 만큼, `CalculationResult` 말고 다른 pydantic model을 모두 허용해줘야 할 것 같아요.

Reply:
https://github.com/eunnbi/zero-to-hero/pull/1/commits/c854b7cfdbc1045e00a339e25cba2609bd8bbf6c